src/interfaces/IBribeInitiative.sol

```diff
@@ -70,6 +74,14 @@ interface IBribeInitiative {
     /// @param _epoch Epoch at which the bribe is deposited
     function depositBribe(uint256 _boldAmount, uint256 _bribeTokenAmount, uint256 _epoch) external;
 
+    /// @notice Take previously deposited BOLD and/or bribe tokens from an epoch in which nobody voted for the
+    ///         initiative and deposit them as bribes into the current epoch. In case there are no present LQTY
+    ///         allocations, and it's no longer possible to allocate LQTY to the initiative because it's been
+    ///         unregistered, the bribes are paid out to the caller instead.
+    /// @param _originalEpoch Epoch at which the bribes were originally deposited
+    /// @param _prevTotalLQTYAllocationEpoch Epoch at which the total allocated LQTY became zero
+    function redepositBribe(uint256 _originalEpoch, uint256 _prevTotalLQTYAllocationEpoch) external;
```

Comment:
I think I would call this something like “recoverOrRedepositBribe”. We may even keep it simple and allow only recovery, and then it would be up to the owner to deposit them again.

Reply:
Renaming it is fine, but I wouldn't let the owner recover bribes, see my other comment.